docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -270,6 +270,20 @@ In addition to the ``namespace.collection_name:version`` format, you can provide
 
 Verifying against ``tar.gz`` files is not supported. If your ``requirements.yml`` contains paths to tar files or URLs for installation, you can use the ``--ignore-errors`` flag to ensure that all collections using the ``namespace.name`` format in the file are processed.
 
+If a collection has been signed by the Galaxy server, the server will provide ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection's contents. You must opt into signature verification by configuring a keyring for ``ansible-galaxy`` to use and providing the path with the ``--keyring`` option.
```

Comment:
I wonder if it's worth putting a subheading here so folks can find/link to this exact spot in the docs?

Reply:
I added the subheading:

```
Signature verification
----------------------
```

Reply:
Good idea.